packages/connect-ui/src/components/InfoPanel.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+
+import styled from 'styled-components';
+
+const Aside = styled.aside`
+    display: flex;
+    flex-flow: column;
+    height: 100%;
+    border-right: 1px solid #e3e3e3;
+    background-color: #f9f9f9;
+    width: 35vw;
+
+    @media (max-width: 639px) {
```

Comment:
How about using [predefined screen queries](https://github.com/trezor/trezor-suite/blob/b3a294677d0ede5d9a4f99ee79a450f79a0a0c83/packages/components/src/config/variables.ts#L21)?

Reply:
not feasibly at the moment, as long as there is still have popup.css (hardcoded output of former less files from old connect). I don't want to do any changes to it if possible. Once it is gone, then of course, lets use it